scripts/hid_configurator/README.rst

```diff
@@ -82,7 +82,20 @@ Complete the following steps:
        In versions earlier than 5.44, the HID device attached by BlueZ could obtain wrong VID and PID values (ignoring values in Device Information Service), which would stop HIDAPI from opening the device.
        In versions earlier than 5.56, the HID device attached by BlueZ might provide incomplete HID feature report on get operation.
 
-   Additionally, to call the Python script on Linux without root rights, install the provided udev rule :file:`99-hid.rules` file by copying it to :file:`/etc/udev/rules.d` and replugging the device.
+#. In addition, if you don't want to use the root access to run the python script, copy the provided udev rule from the :file:`99-hid.rules` file to the :file:`/etc/udev/rules.d` and reconnect the device.
+#. If you want to connect to a device with a different Vendor or Product ID other than the one specified in the file, you can either run the script with the root permission or complete the following step without root permission:
+
+    a. Add a new entry to the :file:`99-hid.rules` file with your Vendor and Product ID.
+    #. Copy the provided udev rule from the :file:`99-hid.rules` file to the :file:`/etc/udev/rules.d`.
+    #. Reconnect the device.
+
+    Vendor and Product ID can be specified in the configuration file related to the nRF Desktop application.
+    The following example shows the entry to add to the :file:`99-hid.rules` file:
+
+    .. parsed-literal::
+       :class: highlight
+
+       ATTRS{idVendor}=="my Vendor ID", ATTRS{idProduct}=="my Product ID", MODE="0666", SYMLINK+="nrf52-desktop-my-app-name"
```

Comment:
Consider adding 2 entries here - for USB and Bluetooth (you can even directly use PID of one of our devices to make it more clear and explicit).

Reply:
I prefer adding with fields witch are supposed to be changed being called my_xyz so that user knows that it is not needed to change other fields like Mode. "Ooo 666 looks bad I will change it to 111 and then, why it doesn't work :("

Reply:
Ok, then we can leave it. User could look at existing file to take a look at an example.

Reply:
`SYMLINK+="nrf52-desktop-my-dev-name"`?